mne_qt_browser/_pg_figure.py

```diff
@@ -390,13 +390,12 @@ def update_data(self):
             data = self.mne.data[self.order_idx]
         else:
             data = self.mne.data[self.range_idx]
+        times = self.mne.times
 
         # Get decim-specific time if enabled
         if self.mne.decim != 1:
-            times = self.mne.decim_times[self.mne.decim_data[self.range_idx]]
+            times = times[::self.mne.decim_data[self.range_idx]]
             data = data[..., ::self.mne.decim_data[self.range_idx]]
-        else:
-            times = self.mne.times
```

Comment:
This "caching" of the decimated times was causing problems with tests locally. Creating views (which slicing does) is a `ns`-order operation:
```
In [1]: import numpy as np
In [2]: x = np.random.RandomState(0).randn(10000)
In [3]: %timeit x[100:1111]
136 ns ± 3.88 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```
So let's not bother, and instead just slice times at the same time / way as the data